components/add_new_cv_form.tsx

```diff
@@ -29,7 +42,16 @@ export default function AddNewCvForm({ id }: Props) {
     certifications: [],
     personal_qualities: [],
     availablePositions: [],
+    cv_skill: [],
   });
+  const [skills, setSkills] = useState({});
+  const [initialUserSkills, setInitialUserSkills] = useState<
+    Array<{
+      id: string;
+      cv_id: string;
+      skill_id: string;
+    }>
+  >([]);
   const [serverErrorMessage, setServerErrorMessage] = useState<string>();
   const [educationsToRemove, setEducationsToRemove] = useState<Array<string>>(
     [],
```

Comment:
We can extract this structure to `SkillResponse` e.g.


```suggestion
    Array<SkillResponse>
```